apps/k8s/ksonnet/myapp/.ksonnet/registries/incubator/422d521c05aa905df949868143b26445f5e4eda5.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: "0.1"
+gitVersion:
+  commitSha: 422d521c05aa905df949868143b26445f5e4eda5
+  refSpec: master
+kind: ksonnet.io/registry
+libraries:
+  apache:
+    path: apache
+    version: master
+  efk:
+    path: efk
```

Comment:
Are all these paths required for this sample? Particularly, `mariadb` and others afterwards.